src/UserSpaceInstrumentation/TrampolineTest.cpp

```diff
@@ -283,17 +284,17 @@ TEST(TrampolineTest, AllocateMemoryForTrampolines) {
 
   // Find the address range of the code for `DoubleAndIncrement`. For the purpose of this test we
   // just take the entire address space taken up by `UserSpaceInstrumentationTests`.
-  AddressRange code_range;
-  auto modules = orbit_object_utils::ReadModules(pid);
-  CHECK(!modules.has_error());
-  for (const auto& m : modules.value()) {
-    if (m.name() == "UserSpaceInstrumentationTests") {
-      code_range.start = m.address_start();
-      code_range.end = m.address_end();
-    }
-  }
+  auto modules_or_error = orbit_object_utils::ReadModules(pid);
+  CHECK(!modules_or_error.has_error());
+
+  auto& modules = modules_or_error.value();
+  const auto module = std::find_if(modules.begin(), modules.end(), [&](const auto& module) {
+    return module.name() == "UserSpaceInstrumentationTests";
+  });
+
+  ASSERT_NE(module, modules.end());
+  const AddressRange code_range{module->address_start(), module->address_end()};
 
-  // Allocate one megabyte in the tracee. The memory will be close to `code_range`.
```

Comment:
Was this deleted by accident?

Reply:
Ah yes. Probably. Let me quickly add this again.